students/rusty_mann/session04/katas.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+import random
+import os
+
+
+def read_in(filename):
+    '''
+    read the contents of a text document from a Project Gutenberg EBook
+    param: name of text file
+    return: one long string containing entire text of book except table of contents
+    '''
+    with open(filename, 'r') as f:
+        for i in range(61):
+            f.readline()
+        text = []
+        for line in f:
+            if line.startswith("End of the Project Gutenberg EBook"):
+                break
+            text.append(line)
+    return " ".join(text)
```

Comment:
Very good, nice catch getting rid of the non-book stuff at the beginning and end.